crime-commons-mod-schemas/src/main/resources/schemas/evidence/common/apiIncomeEvidence.json

```diff
@@ -0,0 +1,33 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "id": "IncomeEvidence.json",
```

Comment:
Does this need to be `apiIncomeEvidence.json` to match the name of file?

Reply:
Good catch - I've pushed a fix now